cmd/acra-connector/acra-connector.go

```diff
@@ -77,44 +80,56 @@ func handleApiConnection(config *Config, connection net.Conn) {
 }
 
 func handleConnection(config *Config, connection net.Conn) {
+	options := []trace.StartOption{trace.WithSpanKind(trace.SpanKindClient)}
```

Comment:
what is "span kind client"?

Reply:
just metadata that indicate client/server side of span
https://opencensus.io/core-concepts/tracing/#spankind
it does not affect anything but show it in some trace collectors